.yarn/patches/react-native-npm-0.68.0-9eb3ecb60a.patch

```diff
@@ -1,8 +1,17 @@
 diff --git a/jest/preprocessor.js b/jest/preprocessor.js
```

Comment:
Yarn placed this file inside `.yarn` directory. Perhaps a good idea?

Reply:
yep, this is where it should be (new in yarn v3 there's a built in `--save`)